plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -42,7 +42,7 @@ fu_usi_dock_mcu_device_tx(FuUsiDockMcuDevice *self,
 			  gsize bufsz,
 			  GError **error)
 {
-	g_autoptr(FuStructUsiDockMcuCmdReq) st = fu_struct_usi_dock_mcu_cmd_req_new();
+	g_autoptr(GByteArray) st = fu_struct_usi_dock_mcu_cmd_req_new();
```

Comment:
@victor-cheng I don't think we want this change -- we changed them to be opaque pointers some months ago.